### PR TITLE
2024-05-16 AOSP merge

### PR DESCRIPTION
**Pull request by Databean**

### CODE REVIEW

File: base/cvd/cuttlefish/host/commands/cvd/server_command/start.cpp
Comment (jemoreira) [inline base/cvd/cuttlefish/host/commands/cvd/server_command/start.cpp:1]: This file shouldn't be modified on an import from AOSP

File: base/cvd/launch_cvd.proto
Comment (jemoreira) [inline base/cvd/launch_cvd.proto:1]: This file looks misplaced.

**Review by jemoreira — CHANGES_REQUESTED** [review]

File: base/cvd/cuttlefish/host/commands/cvd/server_command/start.cpp
Comment (Databean) [reply base/cvd/cuttlefish/host/commands/cvd/server_command/start.cpp:1]: `RecursivelyRemoveDirectory` changed signature, it used to return `bool` but now returns `Result<void>`.

File: base/cvd/launch_cvd.proto
Comment (Databean) [reply base/cvd/launch_cvd.proto:1]: Looks like PR #536 didn't update the merge path for this file.

**Review by jemoreira — APPROVED** [review]

**Comment by Databean** [comment]
Thanks for reviewing!

**Pull request merged**